src/main/webapp/tests/unit/components/MultipleSelect.spec.js

```diff
@@ -15,25 +15,25 @@ describe('MultipleSelect.vue', () => {
         // expect(emitted().input.length).toEqual(5);
     });
 
-    function setupComponent() {
-        return render(
-            MultipleSelect,
-            {
-                propsData: {
-                    selectedCategories: [],
-                    categories: [
-                        'fontend',
-                        'backend',
-                        'css',
-                        'javascript',
-                        'vue',
-                        'java'
-                    ]
-                }
-            },
-            localVue => {
-                localVue.component('v-select', vSelect);
-            }
-        );
-    }
+    //     function setupComponent() {
+    //         return render(
+    //             MultipleSelect,
+    //             {
+    //                 propsData: {
+    //                     selectedCategories: [],
+    //                     categories: [
+    //                         'fontend',
+    //                         'backend',
+    //                         'css',
+    //                         'javascript',
+    //                         'vue',
+    //                         'java'
+    //                     ]
+    //                 }
+    //             },
+    //             localVue => {
+    //                 localVue.component('v-select', vSelect);
+    //             }
+    //         );
+    //     }
```

Comment:
If this is not needed then please remove it

Reply:
ok that was supposted to be todo for christoph